report/sections/system-design.tex

```diff
@@ -34,10 +35,205 @@ \subsection{Model}
+\begin{figure}[h!]
+\centering
+\includegraphics[width=0.4\textwidth]{sections/assets/car-unicycle.png}
+\caption{Unicycle model of a car-like robot. 
+$v_L$ and $v_R$ represent the left- and right-hand side wheels' velocities respectively. 
+The robot follows a path around the instantaneous center of rotation (ICR) where $R_L$ and $R_R$ are the distances from left and right wheels to ICR respectively and $w$ is the distance between them.}
+\label{fig:UnicycleModel}
+\end{figure} 
+
+As shown in Fig.~\ref{fig:UnicycleModel} The robot follows a curved path with the instantaneous center of rotation at its center. 
+The left-hand side wheels have velocity $v_L$ and moves along an arc with radius $R_L$ during the time that right-hand side wheels moves along another arc with radius $R_R$ at the speed of $v_R$. 
+The turning rate of the body is
+\begin{equation*}
+\dot{\theta}= \frac{v_L}{R_L} = \frac{v_R}{R_R}
+\end{equation*}
+
+since $R_R = R_L + W$ the expression can be simplified as
+\begin{equation}
+\dot{\theta}= \frac{v_R - v_L}{W} = \frac{v_\Delta}{W}\label{eq:ThetaDot}
+\end{equation}
+
+the equations of motion for this model are
+\begin{eqnarray}
+\begin{aligned}
+\dot{x} &= v\,cos(\theta)\\
+\dot{y} &= v\,sin(\theta)\\
+\dot{\theta} &= \omega = \frac{v_\Delta}{W}
+\end{aligned}
+\label{eq:MotionEq}
+\end{eqnarray}
+
+where the average velocity\parencite{Corke2011} is given by        
+\begin{equation}
+v = \frac{v_R + v_L}{2} 
+\label{eq:av_velocity}
+\end{equation}
 
 \subsection{Simulation}
 % Simulte the models from the previous section and show that it will work.
 % Motivate regulation approach.
+\subsubsection{Mobile platform}
+A simulation profile is made to make sure of the unicycle mode in sub section \ref{subsubsec:unicycle} is the suitable model for this application.
+The simulation is made in discrete time domain with the sampling frequency of $10Hz$.
+This is done in the described way due to the robot is going to operate in discrete time.
+The sampling frequency is assumed to be low and that is because the designed controllers are going to be able to work fine for higher samplings frequencies but that is not guaranteed for vise-versa.\\  
+As the unicycle model describes the systems dynamics.
+The position of the robot $(x, y, \theta)$ is dependent on the two wheel velocities. Which in turn has dependencies to the average velocity $V$ and the angular velocity $\omega$. These relations are presented in Eq.(\ref{eq:ThetaDot}) - Eq.(\ref{eq:av_velocity}).\\
+\begin{figure*}[ht]
+\centering
+\includegraphics[width=\textwidth]{sections/assets/omegaCtrlr.png}
+\caption{Overview of the systems block diagram where the angular velocity $\omega$ is controlled by a PID controller.}
+\label{fig:overview}
+\end{figure*} 
+
+A pole placement analyze is required for the design of a PID controller.
+This analyze is done in continues time domain, due to its simplicity.\\
```

Comment:
Don't specity `\\` (linebreak) here or in other paragraphs. Paragraph typesetting is better left to LaTeX itself. If it looks bad/weird without them we'll do a once-over when finalizing the report and handle any manual line breaks if needed then.

Reply:
should I remove all \\ (linebreak) in the section?

Reply:
Remove all `\\` in the file.